doc/api/assert.markdown

```diff
@@ -87,9 +87,9 @@ Custom error validation:
 
 ## assert.doesNotThrow(block[, message])
 
-Expects `block` not to throw an error, see `assert.throws` for details.
+Expects `block` not to throw an error. See `assert.throws()` for details.
 
 ## assert.ifError(value)
 
-Tests if value is not a false value, throws if it is a true value. Useful when
-testing the first argument, `error` in callbacks.
+Tests if value is an `Error` and, if so, throws it. This is useful when
```

Comment:
The original text is correct, just kind of worded in a way that's easy to misunderstand. `throws if it is a true value` makes it sound like it throws an error if the value is true. But what it actually does is it throws the value itself, as long as it is truthy. I'll attempt a re-wording that won't make it WRONG...